tests/api/test_account_utils.py

```diff
@@ -517,3 +503,108 @@ def test_get_not_manageable_permissions_after_group_deleting_some_cannot_be_mana
         group1
     )
     assert non_managable_permissions == {"order.manage_orders"}
+
+
+def test_get_not_manageable_permissions_removing_users_from_group(
+    staff_users, permission_group_manage_users, permission_manage_staff
+):
+    """Ensure not returning permission when some of users stay in group and groups has
+    manage staff permission.
+    """
+    group = permission_group_manage_users
+    group.permissions.add(permission_manage_staff)
+    group.user_set.add(*staff_users)
+
+    permissions = get_not_manageable_permissions_after_removing_users_from_group(
+        group, staff_users[1:]
+    )
+
+    assert not permissions
+
+
+def test_get_not_manageable_perms_removing_users_from_group_some_user_from_group_can(
```

Comment:
`users_from_group_some_user_from_group_can` :confused:

Reply:
I changed the name a little. It's too long to create something more descriptive.